backend/test/removeReported.test.js

```diff
@@ -0,0 +1,48 @@
+const removeReported = require('../removeReported');
+
+describe('getting stored reported users', () => {
+
+    console.log("no email")
+    var resultArray = removeReported("")
+    test('array has correct length when no gmail is given', () => {
+        expect (resultArray).toHaveLength(7);
+    });
+    test('array has correct content when no gmail is given', () => {
+        expect (resultArray[0]).toBe("aaa@gmail.com");
+        expect (resultArray[1]).toBe("bbb@gmail.com");
+        expect (resultArray[2]).toBe("ccc@gmail.com");
```

Comment:
Codacy has a fix for the issue: [Replace `········expect·` with `····expect`](https://app.codacy.com/gh/karatwan/CPEN321_world_resources/pullRequest?prid=9811192)
```suggestion
    expect(resultArray[2]).toBe("ccc@gmail.com");
```